File: src/test/unit/AdvisorTest.java
package test.unit;

import static org.junit.Assert.assertEquals;

import java.util.ArrayList;
import java.util.Calendar;
import java.util.GregorianCalendar;
import java.util.List;

import org.junit.Test;

import Debt_LRA_Transcript.LectureRegistrationApplication;
import Enums.InstructorType;
import IDs.InstructorID;
import lecture.Lecture;
import person.Advisor;
import person.Student;

public class AdvisorTest {
	@Test
	public void test_advisor_list_of_student() {
		
		InstructorID testInstructorID = new InstructorID(1,2);
		List<Lecture> testListOfLectures = new ArrayList<Lecture>();
		Calendar testDateOfEntry = new GregorianCalendar(2013,10,28);
		List<Student> testListOfStudents = new ArrayList<Student>();
		List<LectureRegistrationApplication> testListOfApplications = new ArrayList<LectureRegistrationApplication>();
		InstructorType testInstructorType = InstructorType.Assistant;
		Advisor testAdvisor = new Advisor("A","B",testInstructorID,testListOfLectures,testDateOfEntry,testListOfStudents,testListOfApplications,testInstructorType);
	
		assertEquals(testListOfStudents, testAdvisor.getListOfStudents());
	}	
		
	@Test
	public void test_advisor_list_of_application() {
		
		InstructorID testInstructorID = new InstructorID(1,2);
		List<Lecture> testListOfLectures = new ArrayList<Lecture>();
		Calendar testDateOfEntry = new GregorianCalendar(2013,10,28);
		List<Student> testListOfStudents = new ArrayList<Student>();
		List<LectureRegistrationApplication> testListOfApplications = new ArrayList<LectureRegistrationApplication>();
		InstructorType testInstructorType = InstructorType.Assistant;
		Advisor testAdvisor = new Advisor("A","B",testInstructorID,testListOfLectures,testDateOfEntry,testListOfStudents,testListOfApplications,testInstructorType);
	
		assertEquals(testListOfApplications, testAdvisor.getListOfApplications());
	}
		
	
	
	
}

File: pythonCode/AdvisorTest.py
#!/usr/bin/env python
from Advisor import Advisor
import datetime
import unittest
class AdvisorTest(unittest.TestCase):
    # generated source for class AdvisorTest 
    def test_advisor_list_of_student(self):
        # generated source for method test_advisor_list_of_student 
        testListOfStudents = list()
        testAdvisor = Advisor(None, None, None, None, None, testListOfStudents, None, None, None)
        self.assertEquals(testListOfStudents, testAdvisor.getListOfStudents())
    def test_advisor_list_of_application(self):
        # generated source for method test_advisor_list_of_application 
        testListOfApplications = list()
        testAdvisor = Advisor(None, None, None, None, None, None, testListOfApplications, None, None)
        self.assertEqual(testListOfApplications, testAdvisor.getListOfApplications())
if __name__ == '__main__':
    unittest.main()